index.js

```diff
@@ -159,6 +160,7 @@ Fluenture.prototype.and = unaryDispatcher (fluent, and);
 //.
 //. Fluent [`ap`](https://github.com/fluture-js/Fluture#ap).
 Fluenture.prototype.ap = unaryDispatcher (fluent, ap);
+Fluenture.prototype['fantasy-land/ap'] = unaryDispatcher (fluent, ap);
```

Comment:
I would like to double-check that the argument order is the right way around (also for `alt`)